app/src/main/java/com/medmapper/v33001/room/MedicineRoomDatabase.kt

```diff
@@ -7,10 +7,10 @@ import androidx.room.RoomDatabase
 import com.medmapper.v33001.dao.MedicineDAO
 import com.medmapper.v33001.dto.Medicine
 
-@Database(entities = arrayOf(Medicine::class), version = 1, exportSchema = false)
+@Database(entities = [Medicine::class], version = 1, exportSchema = false)
 public abstract class MedicineRoomDatabase : RoomDatabase() {
 
-    abstract fun medicineDao(): MedicineDAO
+    abstract suspend fun medicineDao(): MedicineDAO
```

Comment:
Suggested added coroutine `suspend ` here, performing this database call asynchronously should help with performance